recipes/trio-asyncio/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "trio_asyncio" %}
+{% set version = "0.7.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0b85aece9250c5db73c22f3a55ed5cd00facdf741d7b672935c1ca88e83f9436" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python 3.6.*
```

Comment:
Why is it pinned to Python 3.6? Should this be a version bound instead or is breakage expected in Python 3.7?

Reply:
If I relaxed it to `>=3.6` I would need a selector to eliminate `contextvars` for py37. Since I'm using `noarch: python` I can't use selectors and since py37 isn't released yet I just pinned it to `3.6.*` for now.

It's easy enough to relax that when py37 is released and maybe by then the noarch selector issue will have been resolved.